configuration.go

```diff
@@ -148,65 +174,14 @@ func loadWhitelistFromCSV(r io.Reader) (mapset.Set, error) {
 
 		orgID, err := strconv.Atoi(line[0]) // single record per line
```

Comment:
just a though: orgID might be unsigned int, not signed one ;)

Reply:
fixed